src/main/java/com/ibm/cloud/cloudant/kafka/mappers/DocumentToSourceRecord.java

```diff
@@ -79,8 +86,11 @@ public SourceRecord apply(String topic, ChangesResultItem changesResultItem) {
         offsetFunction.apply(changesResultItem.getSeq()),
         topic,
         RECORD_KEY_SCHEMA,
-        Collections.singletonMap(CloudantConst.CLOUDANT_DOC_ID, changesResultItem.getId()),
-        RECORD_VALUE_SCHEMA,
+        new Struct(RECORD_KEY_SCHEMA)
+                .put(CloudantConst.CLOUDANT_DOC_ID, changesResultItem.getId())
+                .put(InterfaceConst.DB, partition.get(InterfaceConst.DB))
+                .put(InterfaceConst.URL, partition.get(InterfaceConst.URL)),
+                RECORD_VALUE_SCHEMA,
```

Comment:
This has the wrong indent now
```suggestion
        RECORD_VALUE_SCHEMA,
```